tests/0000-unittests.c

```diff
@@ -27,7 +27,7 @@
  */
 
 #include "test.h"
-#include "rdkafka.h"
+#include <librdkafka/rdkafka.h>
```

Comment:
Don't see a reason for having any of the tests to change since they'll never be built outside the source tree.

Reply:
Yeah, your're right. I'm gonna remove this.